dev-python/matplotlib2tikz/matplotlib2tikz-0.6.6.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Convert matplotlib figures into TikZ/PGFPlots"
+HOMEPAGE="https://github.com/nschloe/matplotlib2tikz"
+SRC_URI="https://github.com/nschloe/matplotlib2tikz/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+PATCHES=( "${FILESDIR}/${PN}-0.6-init_pipdated.patch" )
+
+RDEPEND="
+	dev-python/matplotlib[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/pillow[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
```

Comment:
@marbre, then add it ;-). You can do it in the same PR (but put the extra commit before the bump) or separately, either way works for me. You can add the Python team as a secondary maintainer for that since it seems like a potentially reusable library.